coffea/nanoevents/transforms.py

```diff
@@ -265,16 +274,16 @@ def nestedindex(stack):
     """
     indexers = stack[:]
     stack.clear()
-    # return awkward1.concatenate([idx[:, None] for idx in indexers], axis=1)
+    # return ak.concatenate([idx[:, None] for idx in indexers], axis=1)
```

Comment:
I believe concatenate for axis=1 is now implemented. Care to try it?

Reply:
tagging this for rc2